Modules/include/StatusAggregator.h

```diff
@@ -115,6 +115,10 @@ namespace ChimeraTK {
     /// Return value of -1 has the special meaning that the input Status's must be all equal, otherwise it must result
     /// in a warning Status.
     int getPriority(StatusOutput::Status status) const;
+
+    /// Allow runtime debugging (FIXME: use Void instead!)
```

Comment:
Note: `Void` is still not properly working in ApplicationCore, hence I have used `int` instead for the time being...